src/components/migrationToV3.stories.mdx

```diff
@@ -24,7 +24,7 @@ npm install --save amazon-chime-sdk-component-library-react@beta amazon-chime-sd
 - Add `MeetingManagerJoinOptions` as a new interface for the `options` parameter of the `MeetingManager.join` method.
 - Add `deviceLabels`, `eventController`, `enableWebAudio`, and `activeSpeakerPolicy` to `MeetingManagerJoinOptions` interface.
 - Remove `MeetingSessionConfiguration` properties from `MeetingProvider` props.
-- Remove `deviceLabels`, `eventController`, `logLevel`, `postLogConfig`, `logger`, `enableWebAudio`, and `activeSpeakerPolicy` from `MeetingProvider` props.
+- Remove `deviceLabels`, `eventController`, `enableWebAudio`, and `activeSpeakerPolicy` from `MeetingProvider` props.
```

Comment:
Why do we remove `logLevel` and all here?

Reply:
I think its my bad, @xuesichao could you help me fix this? Since you will open PR for release-3.0.0 branch as well.